src/components/Admin/Admin.js

```diff
@@ -0,0 +1,134 @@
+import {React, Component} from 'react';
+import {Table, Tag, Space} from 'antd';
+
+/*Placeholder data*/
+const columns = [
+    {
+      title: 'First Name',
+      dataIndex: 'fname',
+      filters: [
+        {
+          text: 'Joe',
+          value: 'Joe'
+        },
+        {
+          text: 'Jim',
+          value: 'Jim'
+        }
+      ],
+      onFilter: (value, record) => record.fname.indexOf(value) === 0,
+      key: 'fname',
+      render: text => <a>{text}</a>,
+    },
+    {
+      title: 'Last Name',
+      dataIndex: 'lname',
+      filters: [
```

Comment:
Would we be able to type into this at all for searches? 

Can we also lock the table headers on scroll so they are always visible?
(Dont want to scroll to the top to change filters)

Reply:
I imagine it would be possible. I will look into that. Thank you for the suggestions.